stumpy/stump.py

```diff
@@ -312,28 +315,35 @@ def _stump(
             QT_odd[0] = QT_first[i]
             D = _calculate_squared_distance_profile(m, QT_odd, μ_Q[i], σ_Q[i], M_T, Σ_T)
 
+        # Ignore distance profile if query mean is np.nan
+        if np.isinf(μ_Q[i]):
+            D[:] = np.inf
```

Comment:
I think there should be a space between this line and the next. This is minor but the two `if` statements seem independent so it would be nice to have visual separation. Again, minor.